src/data/matrix/basic.lean

```diff
@@ -934,6 +936,12 @@ lemma minor_mul_equiv [semiring α] {p q : Type*} [fintype p] [fintype q]
   (M ⬝ N).minor e₁ e₃ = (M.minor e₁ e₂) ⬝ (N.minor e₂ e₃) :=
 minor_mul M N e₁ e₂ e₃ e₂.bijective
 
+lemma mul_minor_one {n' o' : Type*} [semiring α] [decidable_eq o] (e₁ : n ≃ o) (e₂ : l ≃ o)
+  (M : matrix m n α) : M.mul ((1 : matrix o o α).minor e₁ e₂) = minor M id (e₁.symm ∘ e₂) :=
+begin
+  sorry,
+end
```

Comment:
Yes, this would be perfect, thanks!

Reply:
It's not clear to me that this lemma requires e₂ to be an equiv - but we can try relaxing it once you have the proof

Reply:
OK, I have insterted a proof. It reqreuis that e<sub>2</sub> is an equiv because it uses `lemma minor_minor`, but you might come up with another proof, of course. If not, I hope we're done.